public/app/features/explore/Explore.tsx

```diff
@@ -297,6 +310,15 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
 
   memoizedGetNodeGraphDataFrames = memoizeOne(getNodeGraphDataFrames);
 
+  renderFlameGraphPanel() {
+    const { queryResponse } = this.props;
+    const dataFrames = queryResponse.series.filter(
```

Comment:
Shouldn't we use `flameGraphFrames` here? Seems bit weird to populate them but then not use them and filter again here.